apps/src/templates/sectionProgress/progressTables/ProgressTableTextCells.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const styles = {
+  text: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '100%',
+    minWidth: '30px'
+  },
+  label: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+    height: '100%',
+    paddingRight: '10px'
+  },
+  group: {
+    display: 'flex',
+    justifyContent: 'space-around',
+    padding: '5px'
+  }
+};
+
+export function ProgressTableTextCell({text}) {
+  return <div style={styles.text}>{text}</div>;
+}
+
```

Comment:
NIT: i'm a fan of no whitespace between component and prop types. i feel like that's what we've been doing elsewhere anyway, but here it would be particularly useful for grouping the two separate components.